pages/pipelines/managing-priorities.md.erb

```diff
@@ -0,0 +1,49 @@
+# Job Prioritization
+
+Jobs are dispatched in the following order
+
+  - priority desc (highest priority -> lowest priority)
+  - scheduled_at asc (oldest jobs, note that jobs inherit "scheduled_at" from pipeline upload jobs, meaning jobs that are pipeline uploaded in an older build will be dispatched before later created builds)
+  - order asc (the order they were uploaded in a pipeline)
+  - internal id asc (used as a tie breaker if all value are the same, meaning older jobs will be dispatched first)
+
+Jobs default to a priority of 0, you can prioritise or deprioritise by using higher or lower numbers.
+
+<video src="https://user-images.githubusercontent.com/14028/122886342-39460a00-d383-11eb-898d-692aa057ffa0.mov"></video>
+
+Here's a very basic example:
+
+```yml
+steps:
+  - command: "will-run-last.sh"
+    priority: -1
```

Comment:
yes, I think I read "highest" right - probably good to clarify and also make sure that people understand negative values are acceptable.
What happens on:
```yaml
priority: .inf
priority: -.Inf
priority: .NAN
priority: 1.123456789
```
?

Reply:
Those aren't valid syntax, you'll get a syntax error. Priority only accepts integer values (or null / omitted).

The field is stored as a signed integer, 32 bit, so -2^31 to 2^31-1 (-4294967296 to 4294967295), defaulting to 0.

Reply:
cool 👍

Reply:
(±2 billion rather than ±4 billion, but hopefully nobody needs priorities going into the billions 🙃)